### PR TITLE
Increasing request timeouts in services

### PR DESCRIPTION
**Pull request by TonyRice**

### CODE REVIEW

**Comment by TonyRice** [comment]
This helps solve some issues with certain services taking too long to respond 20+ seconds.

File: storyruntime/processing/Services.py [hunk 1]
```diff
@@ -483,9 +483,15 @@ def smart_insert(cls, story, line, command_conf: dict, key: str, value,
...
         method = command_conf['http'].get('method', 'post')
+
+        # set the default request timeout
+        # to 2 minutes, and set the retry
+        # timeout to 1s
         kwargs = {
             'method': method.upper(),
-            'headers': header_params
+            'headers': header_params,
+            'request_timeout': 60000 * 2,
```
Comment (adnrs96) [inline storyruntime/processing/Services.py]: This should be 60*2
I believe asyncio.sleep accepts seconds
https://docs.python.org/3/library/asyncio-task.html#sleeping

Comment (TonyRice) [reply storyruntime/processing/Services.py]: it's a bit confusing to look at but, request_timeout is passed to AsyncHTTPClient which requires ms, the retry_timeout is what's passed to sleep

Comment (adnrs96) [reply storyruntime/processing/Services.py]: ahh, I misread a bit

**Pull request merged**